apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -111,26 +158,105 @@ class PassengerForm extends React.Component<Props, State> {
     this.setState({ id });
   };
 
+  validateYear = (year: number): boolean => {
+    return year > getYear(new Date()) - 110 && year < getYear(new Date()) - 18;
```

Comment:
Passenger can be also a child or infants. 
By the way, can you split this PR on two?
1. component
2. validation

Thanks

Reply:
Oops, forgot about this

Will try!

Reply:
I don't want to be a pain in the ass so If you want to keep it in one I will be OK with that. I was just confused with the setState at the end of the validation method. So I think that there must be a better way how to handle it.

Reply:
removed setState from validation as its not needed anymore

I could totally divide it, this is what I wanted initially. But the approach of doing it changed several times and at some point I couldnt really divide it. Although now it would be fine.
Sorry for the mess 😞